osi_environment.proto

```diff
@@ -324,4 +336,150 @@ message EnvironmentalConditions
+        // The total degree of coverage indicates how much of the sky is covered with clouds altogether.
+        // 0 oktas means that there are no traces of clouds in the sky, \c
+        // 8 oktas means that the sky is completely covered with clouds and no sky blue can be recognized.
+        //
+        // \par References:
+        // [1] CIE engl. International Commission on Illumination. (2020). <em>CIE S017:2020 ILV: Intl. Lighitng Vocabulary, 2nd edn.</em>. Retrieved March 8, 2022, from https://cie.co.at/eilvterm/17-29-116
+        // [2] UBC The University of British Columbia. (2018). <em>ATSC 113 Weather for Sailing, Flying & Snow Sports</em>. Retrieved March 8, 2022, from https://www.eoas.ubc.ca/courses/atsc113/flying/met_concepts/01-met_concepts/01c-cloud_coverage/index.html
+        //
+        enum FractionalCloudCover
+        {
+            // Fractional cloud cover level is unknown (must not be used in ground truth).
+            //
+            FRACTIONAL_CLOUD_COVER_UNKNOWN = 0;
+
+            // Other (unspecified but known) fractional cloud cover level.
+            //
+            FRACTIONAL_CLOUD_COVER_OTHER = 1;
+
+            // 0/8 of the sky is covered with clouds.
+            //
+            FRACTIONAL_CLOUD_COVER_ZERO_OKTAS = 2;
+
+            // 1/8 of the sky is covered with clouds.
+            //
+            FRACTIONAL_CLOUD_COVER_ONE_OKTAS = 3;
+
+            // 2/8 of the sky is covered with clouds.
+            //
+            FRACTIONAL_CLOUD_COVER_TWO_OKTAS = 4;
+
+            // 3/8 of the sky is covered with clouds.
+            //
+            FRACTIONAL_CLOUD_COVER_THREE_OKTAS = 5;
+
+            // 4/8 of the sky is covered with clouds.
+            //
+            FRACTIONAL_CLOUD_COVER_FOUR_OKTAS = 6;
+
+            // 5/8 of the sky is covered with clouds.
+            //
+            FRACTIONAL_CLOUD_COVER_FIVE_OKTAS = 7;
+
+            // 6/8 of the sky is covered with clouds.
+            //
+            FRACTIONAL_CLOUD_COVER_SIX_OKTAS = 8;
+
+            // 7/8 of the sky is covered with clouds.
+            //
+            FRACTIONAL_CLOUD_COVER_SEVEN_OKTAS = 9;
+
+            // 8/8 of the sky is covered with clouds.
+            //
+            FRACTIONAL_CLOUD_COVER_EIGHT_OKTAS = 10;
+
+            // Sky obscured, describes situations where the sky is not perceivable, e.g. in dense fog.
+            //
+            FRACTIONAL_CLOUD_COVER_SKY_OBSCURED = 11;
+        }
+    }
+
+    //
+    // \brief Defines wind properties.
+    //
+    message Wind
+    {
+        // The origin direction of the wind (not the target direction) in the ground/xy-plane of the \c
+        // world coordinate system. Corresponds to the heading/yaw angle, counted counterclockwise. \c
+        // 0 pointing north. If north not explicitly defined via proj4 reference in ground truth, \c
+        // then north is pointing in y-axis direction. The default orientation (x, y, z) is easting, northing, up [1].
```

Comment:
The '\c's should be removed. (I will do a final clean-up commit)
"If north is not explicitly defined ..."